src/ctia/stores/es/store.clj

```diff
@@ -40,6 +40,10 @@
              [verdict :as ve]]
             [schema.core :as s]))
 
+(defn delete-state-indexes [{:keys [conn index config]}]
+  (when conn
+    (es-index/delete! conn (str index "*"))))
```

Comment:
It's probably better to do a call per store that explicitly lists the indices that we want to remove.  It's dangerous to do it this way, for example, as it would needlessly clear our kibana indices.  Even if we never use it in production, it's still kind of overkill.